.env.example

```diff
@@ -31,6 +31,7 @@ TWILIO_API_KEY=
 TWILIO_APPLICATION_SID=
 TWILIO_AUTH_TOKEN=
 TWILIO_MESSAGE_SERVICE_SID=
+TWILIO_PER_CAMPAIGN_MESSAGING_SERVICE=false
```

Comment:
Rename this to `EXPERIMENTAL_TWILIO_MANUAL_CAMPAIGN_MESSAGING_SERVICES` and use it to conditionally render the UI added in this PR.